packages/ckeditor5-font-mapper/src/fontMapping/FontReplacer.ts

```diff
@@ -0,0 +1,174 @@
+import Node from "@ckeditor/ckeditor5-engine/src/view/node";
+import Element from "@ckeditor/ckeditor5-engine/src/view/element";
+import Text from "@ckeditor/ckeditor5-engine/src/view/text";
+import UpcastWriter from "@ckeditor/ckeditor5-engine/src/view/upcastwriter";
+import DocumentFragment from "@ckeditor/ckeditor5-engine/src/view/documentfragment";
```

Comment:
As some of these clash with lib.dom (which really isn't nice), we may want to rename conflicting imports instead. This also makes it easier reading the code:

```suggestion
import ViewNode from "@ckeditor/ckeditor5-engine/src/view/node";
import ViewElement from "@ckeditor/ckeditor5-engine/src/view/element";
import ViewText from "@ckeditor/ckeditor5-engine/src/view/text";
import UpcastWriter from "@ckeditor/ckeditor5-engine/src/view/upcastwriter";
import ViewDocumentFragment from "@ckeditor/ckeditor5-engine/src/view/documentfragment";
```